go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/ONSdigital/dp-static-file-publisher
 
-go 1.22
+go 1.23
 
 replace (
 	// for linter warning in CI
```

Comment:
This whole section needs to be removed as we should no longer be using the replace statements.  The libraries should be updated in the "require" section and if we're unable to resolve the audit failure then use an exclude statement.  Please see https://github.com/ONSdigital/dp-observation-extractor/blob/develop/go.mod for an example